cluster/manifests/flannel/daemonset.yaml

```diff
@@ -131,9 +108,6 @@ spec:
       - name: flannel-cfg
```

Comment:
All references to `flannel-cfg` can also be removed, can't they?

Reply:
Ah, good catch. Yes.

Reply:
Or at least you need to change its type away from `configMap` to something like `hostPath`. There should be other containers doing it already where you can peek.

Reply:
Because you need to mount the files from the host into container (previously this was done via mounting the configmap).

Reply:
Nevermind, no need to touch the host file system. That's the whole point 🤦  But the references to the configmap we can delete I guess.

Reply:
Yes, deleted in the next commit. Thanks.